TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -212,7 +212,11 @@ public static function initializeErrorHandling(Bootstrap $bootstrap)
 
         $errorHandler = new \TYPO3\Flow\Error\ErrorHandler();
         $errorHandler->setExceptionalErrors($settings['error']['errorHandler']['exceptionalErrors']);
-        $exceptionHandler = new $settings['error']['exceptionHandler']['className'];
+        if (class_exists($settings['error']['exceptionHandler']['className'])) {
+            $exceptionHandler = new $settings['error']['exceptionHandler']['className'];
+        } else {
+            $exceptionHandler = new \TYPO3\Flow\Error\ProductionExceptionHandler();
```

Comment:
Well, is the logging set up at that point? Would need to check, but I guess it's not.